### PR TITLE
Release Notes for CIS v2.5.0

### PR DESCRIPTION
**Pull request by srivastavnitin24**
**Description**:  Added the new functionality introduced and bug fixes for CIS v2.5.0

**Changes Proposed in PR**:

**Fixes**: resolves #_Github issue id_

## General Checklist
- [x] Updated Added functionality/ bug fix in release notes
- [x] Added examples for new feature
- [x] Updated the documentation

## CRD Checklist
- [x] Updated required CR schema

### CODE REVIEW

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -1,38 +1,55 @@
...
+
+
+Bugs Fixes
+``````````
+* CIS now properly adds nodes as pool members (in NodePort mode).
+
+
+Known Issues
+````````````
+* Recommendation to configure CIS deployment with --periodic-sync-interval more than 300 seconds for improved performance. OpenShift Routes with termination Passthrough get processed post this interval.
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:44]: For improved performance, it is recommended to configure CIS deployment with --periodic-sync-interval​more than 300 seconds. OpenShift Routes with termination Passthrough are processed post this interval.

File: docs/RELEASE-NOTES.rst [hunk 2]
```diff
@@ -1,38 +1,55 @@
...
+
+
+F5 IPAM Controller v0.1.4 Release Notes:
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:47]: F5 IPAM Controller v0.1.4

File: docs/RELEASE-NOTES.rst [hunk 3]
```diff
@@ -1,38 +1,55 @@
...
-* Helm Chart Enhancements:
-   - Added ingressClass resource installation with charts
-* Added support for AS3 3.28
+* CIS now compatible with:
+    - Kubernetes 1.21
+    - OpenShift 4.7.13 with OpenShift SDN
+    - AS3 3.28
 
+* Added support for:
+   - :issues: `1824` Ingress and IngressClass objects in networking.k8s.io/v1. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:15]: We can add :

With networking.k8s.io/v1 ingress, Add multiple BIGIP SSL client profiles with annotation "virtual-server.f5.com/clientssl". Refer for Examples <>

Comment (srivastavnitin24) [reply docs/RELEASE-NOTES.rst]: Done

**Review by nandakishorepeddi — APPROVED** [review]
Looks Good

File: docs/RELEASE-NOTES.rst [hunk 4]
```diff
@@ -1,38 +1,51 @@
...
+* New CIS deployment configuration options:
+  - --periodic-sync-interval - Configure the periodic sync of resources.
+  - --hubmode:true - Enable Support for ConfigMaps to monitor services in same and different namespaces.
+  - --disable-teems=true|false - Configure to send anonymous analytics data to F5 TEEM infrastructure.
 
-* Fixed http redirect not working when used virtual-server.f5.com/rewrite-target-url annotation with routes
-* Fixed inconsistent behaviour with CIS (Transport Server) while adding/deleting a node to cluster.
+Bugs Fixes
+``````````
+* :issues: `1824` Ingress and IngressClass objects in networking.k8s.io/v1. Refer for `Examples <https://github.com/F5Networks/k8s-bigip-ctlr/tree/master/docs/config_examples/ingress/networkingV1>`_.
```
Comment (trinaths) [inline docs/RELEASE-NOTES.rst:32]: Not a BUG but an RFE. Add more info - Update CIS ClusterRole to monitor networking.k8s.io/v1 with ingressClass resources and install ingressClass for CIS.

**Review by agupta49 — APPROVED** [review]

**Pull request merged**